cmd/root.go

```diff
@@ -61,6 +61,8 @@ func Execute(ctx context.Context, version string) {
 	rootCmd.Flags().StringArrayVarP(&input.containerCapAdd, "container-cap-add", "", []string{}, "kernel capabilities to add to the workflow containers (e.g. --container-cap-add SYS_PTRACE)")
 	rootCmd.Flags().StringArrayVarP(&input.containerCapDrop, "container-cap-drop", "", []string{}, "kernel capabilities to remove from the workflow containers (e.g. --container-cap-drop SYS_PTRACE)")
 	rootCmd.Flags().BoolVar(&input.autoRemove, "rm", false, "automatically remove container(s)/volume(s) after a workflow(s) failure")
+	rootCmd.Flags().StringArrayVarP(&input.throughAction, "through-action", "", []string{}, "If you are using GitHub Enterprise Server and allow specified actions from GitHub (github.com), you can set actions on this. (e.g. --through-action=github/super-linter)")
+	rootCmd.Flags().StringVar(&input.throughActionToken, "through-action-token", "", "If you are using through-action and you want to use private actions on GitHub, you have to set personal access token")
```

Comment:
I prefer something like `--github-com-action`, **this is my opinion other maintainer may disagree.**
_I don't really like `bypass-ghe-action`._

Reply:
Can I use `--actions-from-github` ?
This PR is to make the actions mapping to `github.com`. 
The bypass flow is implement GHE infra on different companies, 
maybe we don't need to add this naming "bypass".

Reply:
ok, I got it